### PR TITLE
Feature/e commerce protos Clara

### PR DESCRIPTION
**Pull request by CRR257**

### CODE REVIEW

File: staff/clara-rufi/e-commerce-protos/index.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
```
Comment (martimalek) [inline staff/clara-rufi/e-commerce-protos/index.html]: You could put a title here, maybe something like "e-commerce".

File: staff/clara-rufi/e-commerce-protos/test.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
```
Comment (martimalek) [inline staff/clara-rufi/e-commerce-protos/test.html]: Here you could also use a title like "e-commerce test".

File: staff/clara-rufi/e-commerce-protos/index.html (hunk 1 above)
Comment (CRR257) [reply staff/clara-rufi/e-commerce-protos/index.html]: Thanks. I have corrected it :)

File: staff/clara-rufi/e-commerce-protos/test.html (hunk 2 above)
Comment (CRR257) [reply staff/clara-rufi/e-commerce-protos/test.html]: Thanks. I changed it :)

**Pull request merged**